modules/crm_tools/crm_tools.libraries.yml

```diff
@@ -0,0 +1,21 @@
+open-iconic:
+  remote: https://github.com/iconic/open-iconic
+  version: "1.1.1"
+  license:
+    name: MIT
+    url: https://github.com/iconic/open-iconic/blob/1.1.1/ICON-LICENSE
+    gpl-compatible: true
+  js:
+    js/open-iconic.drupal.js: {}
+  css:
+    component:
+      css/open-iconic.drupal.css: {}
+  dependencies:
+    - core/jquery
+
+admin-forms:
```

Comment:
This isn't actually admin forms, but specifically roles stuff? Let's name it as such.